R/cyprus_readutils.R

```diff
@@ -0,0 +1,253 @@
+#' @title HDF5 key for Cyprus CalcLoops scalar-type loops
+#' @description Generates an HDF5 key (full path) for the scalar
+#'              type loops from the Cyprus CalcLoops application.
+#' @param istoch Integer, index of the stochastic sample that the key should
+#'               be generated for.
+#' @param loop_type String, name of loop type. Allowed values:
+#'                  'Scalar', 'dOp'
+#' @param cid Integer, configuration number, internally produced by the CalcLoops
+#'            tool via the "trajectory" input flag. The default is '4' as this is
+#'            often not used in practice.
+#' @export
+cyprus_make_key_scalar <- function(istoch, loop_type, cid = 4){
```

Comment:
does this function and the following ones need export?

Reply:
I guess not, one could leave it out.

Reply:
Done.